phpunit.xml.dist

```diff
@@ -29,6 +30,7 @@
 		<directory>tests/Downloader</directory>
 		<file>tests/File/AwsFileTest.php</file>
 		<file>tests/Backend/Workspaces/ReadOnlyUserTest.php</file>
+		<file>tests/Common/BranchBucketsTest.php</file>
```

Comment:
Jo to tu trochu blbe videt ale je to v `paratest-common`

<img width="545" alt="Screenshot 2021-03-16 at 20 14 16" src="https://user-images.githubusercontent.com/6448364/111366853-5ab49d00-8694-11eb-8cbb-075da4eb98bf.png">

Reply:
Jen ještě checkni že paratest-common projekt má nastavenou featuru pro dev prod mode.

Reply:
Pridal som to pre ten paratest screen je v connection PR